src/Parser.php

```diff
@@ -1580,7 +1592,14 @@ protected function whitespace()
                 } else {
                     $comment[] = $c;
                     $staticComment = substr($this->buffer, $startCommentCount, $endCommentCount - $startCommentCount);
-                    $this->appendComment([Type::T_COMMENT, $staticComment, [Type::T_STRING, '', $comment]]);
+                    $commentStatement = [Type::T_COMMENT, $staticComment, [Type::T_STRING, '', $comment]];
+
+                    list($line, $column) = $this->getSourcePosition($startCommentCount);
+                    $commentStatement[self::SOURCE_LINE] = $line;
+                    $commentStatement[self::SOURCE_COLUMN] = $column;
+                    $commentStatement[self::SOURCE_INDEX] = $this->sourceIndex;
```

Comment:
This ensure that errors happening in comments can be associated to the right position in the source file. I'm doing that only for cases that contain an interpolation (static comments are in the other branch of the conditions) as static comments won't need it (there is nothing that can trigger an error in them).